.github/labels.yml

```diff
@@ -75,6 +90,18 @@
 - name: "Status: ⚠️ triage"
   description: "A label used to categorize issues for prioritization and organization."
   color: "6C757D"
+- name: "Status: 🕐 pending"
+  description: "Issue is penging list."
+  color: "987743"
+- name: "Status: 🛟 help wanted"
+  description: "Help wanted."
+  color: "FAC7E9"
+- name: "Status: 📦 closed"
+  description: "Issue is closed with some resolution."
+  color: "A2C4C9"
```

Comment:
Why is it necessary? Because there is an issue status (closed, opened)

Reply:
It important for statistics and some future automatisation. Why not?